syntax = 'proto3';

message LoginRequest {
    string username = 1;
    string password = 2;
}

message LoginResponse {
    bool success = 1;
    string error_message = 2;
}
